### PR TITLE
refactor: fixing linters in whole project

### PR DESCRIPTION
**Pull request by eapg**
* pylint pre-commit hook wasnt running in whole project. 
* fixing linters in whole project.

### CODE REVIEW

File: .pylintrc [hunk 1]
```diff
@@ -4,7 +4,12 @@ disable=
     C0115, # missing-class-docstring
     C0116, # missing-function-docstring
     R0903, # Too few public methods (0/2) (too-few-public-methods)
+    W0105, # String statement has no effect
     W0221, # Parameters differ from overridden
+    W0237, # Parameter has been renamed to.. in overridden
```
Comment (reynoldmorel) [inline .pylintrc:9]: Why did you need all these statements?

File: src/tests/core/engine/processors/kichen_simulator_integration_test_v2.py [hunk 2]
```diff
@@ -75,21 +75,24 @@ def test_assign_orders_to_available_chefs(self):
         )
 
         order_1 = build_order(order_id=1, status=OrderStatus.NEW_ORDER)
-        self.kitchen_simulator.mongo_order_status_history_controller.set_next_status_history_by_order_id(
+        kitchen_sim = self.kitchen_simulator
```
Comment (reynoldmorel) [inline src/tests/core/engine/processors/kichen_simulator_integration_test_v2.py:78]: Why don't you name it `kitchen_simulator`?

File: src/utils/inventory_ingredient_util.py [hunk 3]
```diff
@@ -40,8 +41,9 @@ def reduce_products_qty_by_ingredients_qty(product_id):
         )
 
     def reduce_products_qty_array_to_final_products_qty_map(product_ids):
+        func = products_qty_array_to_final_products_qty_map_reducer
```
Comment (reynoldmorel) [inline src/utils/inventory_ingredient_util.py:44]: Why don't you name it `reducer`?

**Review by reynoldmorel — CHANGES_REQUESTED** [review]

File: .pylintrc [hunk 4]
```diff
@@ -4,7 +4,12 @@ disable=
     C0115, # missing-class-docstring
     C0116, # missing-function-docstring
     R0903, # Too few public methods (0/2) (too-few-public-methods)
+    W0105, # String statement has no effect
```
[the remark this answers is not in the document]
Comment (eapg) [reply .pylintrc:7]: This warning appears every time you use a paragraph as documentation outside a class. pylint documentation
recommend disable it  if you are doing that. [https://github.com/eapg/EA_RESTAURANT/blob/2ced1f8fe8e10225d5e284ffabc4c91ea2be494e/src/constants/order_status.py#L5](url)

File: .pylintrc (hunk 1 above)
Comment (eapg) [reply .pylintrc:9]: i disabled this because in the generic repository i created methods with parameters like:
get_by_id(obj_id):
.......
and then when i inherit the method from generic i overridden obj_id by produt_id, chef_id, order_id etc, depending the repository.

File: src/core/order_manager.py [hunk 5]
```diff
@@ -51,6 +51,6 @@ def clean_queues_with_full_storage(self, limit_value_before_clean=500):
         for status in [OrderStatus.COMPLETED.name, OrderStatus.CANCELLED.name]:
             if self.get_queue_size(status) > limit_value_before_clean:
                 for _ in range(100):
-                    deque_id = self.get_queue_from_status(status)
+                    _deque_id = self.get_queue_from_status(status)
```
[the remark this answers is not in the document]
Comment (eapg) [reply src/core/order_manager.py:54]: this is the way how pylint ignore unused variables.

File: src/lib/entities/mongo_engine_odm_mapping.py [hunk 6]
```diff
@@ -1,7 +1,9 @@
-from mongoengine import Document, IntField, DateTimeField, StringField
+from mongoengine import Document, IntField, DateTimeField, StringField, QuerySetManager
 
 
 class BaseEntity(Document):
+
+    objects = QuerySetManager()
```
[the remark this answers is not in the document]
Comment (eapg) [reply src/lib/entities/mongo_engine_odm_mapping.py:6]: this is the way how pylint recognize .objects as part of mapping odm entity when we query it.

File: src/tests/core/engine/processors/kichen_simulator_integration_test_v2.py (hunk 2 above)
Comment (eapg) [reply src/tests/core/engine/processors/kichen_simulator_integration_test_v2.py:78]: done.

File: src/utils/inventory_ingredient_util.py (hunk 3 above)
Comment (eapg) [reply src/utils/inventory_ingredient_util.py:44]: done.

File: .pylintrc (hunk 4 above)
[the remark this answers is not in the document]
Comment (reynoldmorel) [reply .pylintrc:7]: Could you move the documentation inside the class if that is not a recommended practice?

File: .pylintrc (hunk 1 above)
Comment (reynoldmorel) [reply .pylintrc:9]: What about the other ones?

File: src/core/order_manager.py (hunk 5 above)
[the remark this answers is not in the document]
Comment (reynoldmorel) [reply src/core/order_manager.py:54]: Why don't you remove this variable?

File: src/lib/entities/mongo_engine_odm_mapping.py (hunk 6 above)
[the remark this answers is not in the document]
Comment (reynoldmorel) [reply src/lib/entities/mongo_engine_odm_mapping.py:6]: Ok

File: .pylintrc (hunk 4 above)
[the remark this answers is not in the document]
Comment (eapg) [reply .pylintrc:7]: done.

File: .pylintrc (hunk 1 above)
Comment (eapg) [reply .pylintrc:8]: done.

File: src/core/order_manager.py (hunk 5 above)
[the remark this answers is not in the document]
Comment (eapg) [reply src/core/order_manager.py:54]: done.

**Review by reynoldmorel — APPROVED** [review]
